src/ui/server/helpers/get-name-email-position-from-owner-and-policy/index.test.ts

```diff
@@ -0,0 +1,71 @@
+import getNameEmailPositionFromOwnerAndPolicy from '.';
+import replaceCharacterCodesWithCharacters from '../replace-character-codes-with-characters';
+import ACCOUNT_FIELD_IDS from '../../constants/field-ids/insurance/account';
+import POLICY_AND_EXPORTS_FIELD_IDS from '../../constants/field-ids/insurance/policy-and-exports';
+import { mockApplication, mockContact } from '../../test-mocks';
+
+const {
+  NAME_ON_POLICY: { NAME, SAME_NAME, OTHER_NAME, IS_SAME_AS_OWNER },
+  DIFFERENT_NAME_ON_POLICY: { POSITION },
+} = POLICY_AND_EXPORTS_FIELD_IDS;
+const { FIRST_NAME, LAST_NAME, EMAIL } = ACCOUNT_FIELD_IDS;
+
+const { owner } = mockApplication;
+
+describe('server/helpers/get-name-email-position-from-owner-and-policy', () => {
+  const firstName = replaceCharacterCodesWithCharacters(owner[FIRST_NAME]);
+  const lastName = replaceCharacterCodesWithCharacters(owner[LAST_NAME]);
+  const email = owner[EMAIL];
+  const position = replaceCharacterCodesWithCharacters(mockContact[POSITION]);
+
+  describe(`when ${SAME_NAME} is provided`, () => {
+    it(`should return object with constructed name and email string, position and ${NAME} set to ${SAME_NAME}`, () => {
+      // @ts-ignore
```

Comment:
Seems we need `as ApplicationPolicyContact` to `mockContact` to avoid having to add `ts-ignore`

Reply:
I will have to do this after this PR since mockContact being used by businessContact